src/sql/workbench/parts/notebook/notebook.component.ts

```diff
@@ -188,19 +188,6 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		this._model.addCell(cellType);
 	}
 
-	// Updates Notebook model's trust details
-	public updateModelTrustDetails(isTrusted: boolean) {
```

Comment:
Moved logic and notification to the model.
- Note that removing the `updateModel` call below and having logic to filter trusted state updates vs. other updates a) saves a text model serialization on trusted state changing, and b) does not force input to be dirty when trust is enabled. These are both good things